src/forum.js

```diff
@@ -60,7 +60,66 @@ const doTopic = () => {
 	}
 }
 
+const doTopicHighlighting = () => {
+	if(!thisPage.forumSection)
+		return;
+		
+	if(thisPage.forumSection.fid == 954) {
+		topics = document.querySelectorAll(".topiclist.topics")[1].children;
+		for(var i = 0; i < topics.length; i++) {
+			highlightTopic(topics[i]);
+		}
+
+	} else if(thisPage.forumSection.shortname = "PP") {
```

Comment:
Bug -- `=` is an assignment operator, therefore this will always evaluate to truthy value.

Reply:
Fixed ;)